src/main_app/admin/routes/errors_route.py

```
@@ -0,0 +1,80 @@
+            selected_file=selected_file,
+            file_content=file_content,
+        )
+
+    def read_text(self, error_file: Path) -> str:
+
+        if not error_file.exists():
+            return "No error log found."
+
+        try:
+            text = error_file.read_text(encoding="utf-8")
+        except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)